tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -114,11 +115,16 @@ def _ParseTranslation(self, translation_body):
 
     translation = {}
 
-    for field_name in translation_body.keys():
-      if isinstance(translation_body[field_name], str):
+    for std_field_name in translation_body.keys():
```

Comment:
are we requiring fully qualified field names in the config?  If not, how are we normalizing?

Reply:
We are expecting the following:
```
CHWS_WDT-17:
  type: HVAC/CHWS_WDT
  id: CDM/123456
  translation:
    return_water_temperature_sensor:
      present_value: "points.foo.present_value"
      ```
      
std_field_name == return_water_temperature_sensor
raw_field_name == foo

Reply:
that name isn't fully qualified though.  (qualified would be /return_water_temperature_sensor).  We generally don't require full qualification in configurations unless it would be ambiguous, but that means the code needs to normalize everything to not get confused (which is why we use [this tuple](https://github.com/google/digitalbuildings/blob/master/tools/validators/ontology_validator/yamlformat/validator/entity_type_lib.py#L34) when referencing fields in types.

Reply:
b/176094783 created will get to it in the next PR while refactoring entity instance to make it more oop

Reply:
updated in the new push